Lib/unittest/mock.py

```diff
@@ -796,8 +798,10 @@ def assert_called_once(_mock_self):
         """
         self = _mock_self
         if not self.call_count == 1:
-            msg = ("Expected '%s' to have been called once. Called %s times." %
-                   (self._mock_name or 'mock', self.call_count))
+            msg = ("Expected '%s' to have been called once. Called %s times.\n"
+                   "Calls %r. " % (self._mock_name or 'mock',
```

Comment:
If you get this into a helper method, I'd remove this when the mock is not called, as it will look weird to see this line with no calls, it would just add noise

Reply:
I concur. If there is no call, just omit "Calls..." ;-)